docs/sources/configure/examples/7-Schema-Migration-Snippet.yaml

```
@@ -1,22 +1,24 @@
 schema_config:
   configs:
-    # Starting from 2018-04-15 Loki should store indexes on Cassandra
+    # Starting from 2018-04-15 Loki should store indexes on BoltDB
     # using weekly periodic tables and chunks on filesystem.
     # The index tables will be prefixed with "index_".
   - from: "2018-04-15"
-    store: cassandra
+    store: boltdb-shipper
     object_store: filesystem
-    schema: v11
+    schema: v12
     index:
         period: 168h
```

```suggestion
        period: 24h
```
boltdb also requires 24h period

---

Thank you! Fixed.